### PR TITLE
Change Shortcut Description Sample

### PR DESCRIPTION
**Pull request by michaelScarfi**
Added sample code to modify a shortcut's description

### CODE REVIEW

File: stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+			//Load and populate the shortcut object
+			final WebShortcut SHORTCUT = getWebShortcut(WEB_OBJECT_SOURCE, SHORTCUT_ID);
+			
+			//Print the current value of the shortcuts description
+	    		System.out.println("Previous Description: " + SHORTCUT.getDescription());
+			
+			//Print the new description and change the shortcut's description to that value.
+	    		System.out.println("Setting new Description to: " + NEW_DESC);
+	    		setShorDesc(WEB_OBJECT_SOURCE, SHORTCUT, NEW_DESC);
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java]: Typo: `setShorDesc` -> `setShortDesc`

File: stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	
+	/**
+     	 * Takes a WebObjectSource object, WebShortcut Object and String desc and changes the Destription of the WebShortcut object to the value of desc
+     	 * 
+     	 * @param objectSource Object of type WebObjectSource
+     	 * @param shortcut Object of type WebShortcut
+	 * @param desc String containing the new Description
+     	 * @throws WebObjectsException
+     	 */
+	private static void setShorDesc(WebObjectSource objectSource, WebShortcut shortcut, String desc) throws WebObjectsException {
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java]: Typo also here

File: stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
+package com.microstrategy.samples.objects.Shortcut;
```
Comment (tiagosiebler) [inline stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java]: Can you rename the folder/package to lower-case `shortcut`, to fit with the other samples in the `objects` folder?

Comment (michaelScarfi) [reply stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java]: I changed the package name in code but I do not see a way to change the folder that contains it.

Comment (michaelScarfi) [reply stand-alone/com/microstrategy/samples/objects/Shortcut/ShortcutDescChange.java]: I changed the folder name as well.

**Pull request merged**